packages/expo-cli/src/commands/upload.js

```diff
@@ -40,6 +41,10 @@ export default program => {
       '--apple-id <apple-id>',
       'your Apple ID username (you can also set EXPO_APPLE_ID env variable)'
     )
+    .option(
+      '--itc-team-id <itc-team-id>',
```

Comment:
Can we unify this with `expo build:ios` and name this flag just `--team-id`?

Reply:
This is not the same team id as in `build:ios` recently apple unified iTunes connect app developer portal teams, but fastlane still requires ITC team.

Reply:
In case I need to help someone use this feature in the future, where does one find the iTunes Connect (now called App Store Connect) team ID? I searched, but couldn't find it on https://appstoreconnect.apple.com.

Reply:
Actually, I couldn't find it either but if you call upload:ios without this flag fastlane will list your teams interactively with those ids. I'm assuming that those ids were removed from App Store Connect after  February 12 (https://developer.apple.com/support/teams/)

Reply:
Maybe we could have a team name flag instead of team ID flag (it would set the [FASTLANE_ITC_TEAM_NAME](https://github.com/fastlane/fastlane/blob/d738d74a1070b8a8eb8ffb69374589d929364ef2/spaceship/lib/spaceship/tunes/tunes_client.rb#L63) environment variable). I believe team names are usually the same between App Store Connect and Apple Developer website?

Reply:
names would solve that problem but it would be problematic for users and probably not portable between different shells (especially escaping spaces)